sds_data_manager/lambda_code/SDSCode/create_schema.py

```diff
@@ -5,14 +5,16 @@
 import sys
 
 import requests
+from SDSCode.database.database import engine
+from SDSCode.database.models import Base
```

Comment:
```suggestion
import requests

from SDSCode.database.database import engine
from SDSCode.database.models import Base
```

Reply:
it seems like pre-commit checks removes that space between `requests` and `engine` lines.